README.md

```diff
@@ -122,6 +122,7 @@ To ease development, you can also use `npm run compile-continuously` to have you
 
 > In this case, you could probably also make use of [`live-server`](https://www.npmjs.com/package/live-server).
 
+Or you can even run `npm run dev` for automatic page reloading on change at the url: `http://localhost:8080/webpack-dev-server/cout-embauche-widget`
```

Comment:
Meh. Same thing, the day we change the name of the widget, we'll forget to update the doc… Well, if there's no way to make `webpack-dev-server` output that, let's got with it.